docs/RELEASE_NOTES.md

```diff
@@ -26,15 +26,25 @@
 
 7. Add Migration guide section which specifies explicitly if there are breaking changes and how to tackle them.
 
+### 3.0.0 - MMM DD, 2020
```

Comment:
This is the problem in `create-shoppinglist-sync-utils-and-helper` when it's fixed there, this diff should be gone.